english/explore-outside/journey-to-flaming-tower/start-running/keep-running/search-and-rescue/search-and-rescue.md

```diff
@@ -0,0 +1,17 @@
+While happily reunited with your girlfriend, 
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.